sidecar2cdxj.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import ast
+import json
+import os
+import re
+
+import surt
+from warcio.archiveiterator import ArchiveIterator
+from warcio.timeutils import iso_date_to_timestamp
+
+
+def create_sidecar_cdxj(sidecar_file, archive_dir):
+    if not os.path.isdir(archive_dir):
+        os.mkdir(archive_dir)
+
+    warc_file = os.path.basename(sidecar_file)
+    cdxj_file = re.sub('warc.meta.gz', 'cdxj', warc_file)
+    sidecar_file_path = os.path.join(archive_dir, cdxj_file)
+
+    with open(sidecar_file_path, 'wt') as out, open(sidecar_file, 'rb') as stream:
```

Comment:
I'm wondering if it is better to write a bytes file rather than text, or if it makes no difference.

Reply:
I did it as text because that's how they do it in webrecorder. https://github.com/webrecorder/warcio/blob/aa702cb321621b233c6e5d2a4780151282a778be/warcio/indexer.py#L26